src/content/basics/multiaz/index.md

```diff
@@ -0,0 +1,56 @@
++++
+title = "Clusters Over Multiple Availability Zones"
+description = "By default, cluster get started within a single availability zone. But you can define how many availability zones a cluster should have to get higher a availability for your clusters."
+date = "2018-11-22"
+weight = 50
+type = "page"
+categories = ["basics"]
++++
+
+# Clusters Over Multiple Availability Zones
+
+*This is currently only available with Giant Swarm 6.1.0 on AWS*
+
+With Giant Swarm you can easily launch cluster across multiple availability zones (AZs). This will lower the risk that your cluster will get unavailable while there is an incident with your cloud providers or your datacenters infrastructure.
```

Comment:
I would also mention that node are distributed evenly through the AZs and you can not choose where a node will be allocated